packages/fluentui/react-bindings/src/FocusZone/FocusTrapZone.tsx

```diff
@@ -140,7 +142,10 @@ export class FocusTrapZone extends React.Component<FocusTrapZoneProps, {}> {
         <ElementType
           {...unhandledProps}
           className={className}
-          ref={this.createRef}
+          ref={(ele: HTMLElement) => {
+            handleRef(this.createRef, ele);
+            handleRef(this.props.innerRef, ele);
+          }}
```

Comment:
```suggestion
          ref={(element: HTMLElement) => {
            this.createRef(element);
            handleRef(this.props.innerRef, element);
          }}
```

I guess that `this.createRef` is just a function, let's simply call it

Reply:
Done